src/renderer/views/User.js

```diff
@@ -22,52 +28,32 @@ class User extends Component {
     render() {
         return (<div>
             <div className="single-channel-image">
-                <img className="img-fluid mh-20" style="object-fit: cover; object-position: center; max-height: 500px;" alt="" src="https://img.3speakcontent.online/user/nicksmitley/cover.png" />
+                <img className="img-fluid mh-20" style={{objectFit: 'cover', objectPosition: 'center', maxHeight: '500px'}} alt="" src="https://img.3speakcontent.online/user/nicksmitley/cover.png" />
```

Comment:
Could you add functionality for generating the channel image dynamically? I have not added support for getting a user's banner image yet through simple API call. However, it is pretty easy to build a work around. Such as
```javascript
src={() => {
var ref = RefLink.parse(this.state.reflink)
switch(ref.source.value) {
  case "hive": {
  return `https://img.3speakcontent.online/user/${ref.root}/cover.png`
  }
}}
```